app/src/main/java/edu/unikom/herbamedjabar/view/ForumFragment.kt

```
@@ -10,26 +10,28 @@ import androidx.recyclerview.widget.LinearLayoutManager
-import edu.unikom.herbamedjabar.R
+
 @AndroidEntryPoint
 class ForumFragment : Fragment() {
 
     private var _binding: FragmentForumBinding? = null
-    private val binding get() = _binding!!
+    private val binding
+        get() = _binding!!
 
     private val viewModel: ForumViewModel by viewModels()
     private lateinit var postAdapter: PostAdapter
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)